src/SimpleMaths.java

```diff
@@ -0,0 +1,21 @@
+public class SimpleMaths {
+
+    public static void main(String[] args) {
+
+        double numberOne = 278;
+        double numberTwo = 192;
+
+        int sum = (int) (numberOne + numberTwo);
+        int difference = (int) (numberOne - numberTwo);
+        int product = (int) (numberOne * numberTwo);
+        double qoutient = numberOne / numberTwo;
```

Comment:
nicht ganz was wir wollten - ich wollte gerne, dass du einmal mit int rechnest damit du u.a. siehst dass bei der integer division die precision (alles nach dem komma) einfach wegfällt (auch wenn du das resultat in einem double speicherst). 

Also insgesamt möchte ich 8 Rechnungen 4 mal pro Datentyp ;) 

Und für die formattierte Ausgabe kannst du printf verwenden